sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -342,14 +347,24 @@ extern "C" {
 #endif
 
 /**
- * @brief Default message queue size to use for the GoMacH thread.
+ * @brief Default message queue size to use for the GoMacH thread (as exponent
+ *        of 2^n).
+ *
+ * As the queue size ALWAYS needs to be power of two, this option represents the
+ * exponent of 2^n, which will be used as the size of the queue.
  *
  * The value of this macro should be enough for supporting the manipulation of
  * GoMacH.
  */
-#ifndef GNRC_GOMACH_IPC_MSG_QUEUE_SIZE
-#define GNRC_GOMACH_IPC_MSG_QUEUE_SIZE        (8U)
+#ifndef CONFIG_GNRC_GOMACH_IPC_MSG_QUEUE_SIZE_EXP
+#define CONFIG_GNRC_GOMACH_IPC_MSG_QUEUE_SIZE_EXP        (3U)
```

Comment:
How about "_LOG2" instead of "_EXP"?

Reply:
I find `EXP` way more clearer than `LOG2`.

Reply:
(Yes `LOG2` is mathematically more correct, but I think more people have a grasp of what the binary exponent for the queue size of 8 is, than what the binary logarithm of queue size 8 is)